src/containers/Tag/TagList/TagList.tsx

```diff
@@ -21,11 +21,16 @@ import { GET_TAGS } from '../../../graphql/queries/Tag';
 import { NOTIFICATION } from '../../../graphql/queries/Notification';
 import { DELETE_TAG } from '../../../graphql/mutations/Tag';
 import { ToastMessage } from '../../../components/UI/ToastMessage/ToastMessage';
+import DialogBox from '../../../components/UI/DialogBox/DialogBox';
 
 export interface TagListProps {}
 
 export const TagList: React.SFC<TagListProps> = (props) => {
   const client = useApolloClient();
+
+  // DialogBox states
+  const [tagID, setTagID] = useState(0);
```

Comment:
Variable names are very confusing. May be changed to showDialog and setShowDialog

Reply:
I'm storing the tagID here.. So I think we should leave it as TagID or something similar and not showDialog.